File: Python/tests/test_ts_par7.py
import pytest

from hbayesdm.models import ts_par7


def test_ts_par7():
    _ = ts_par7(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_ts_par7.R
context("Test ts_par7")
library(hBayesDM)

test_that("Test ts_par7", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(ts_par7(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
